test-e2e/ajs.test.ts

```diff
@@ -2,13 +2,12 @@ Feature('AJS Bundle');
 
 const assert = require('assert');
 const testSite = 'https://www.library-test-site.com';
-const testWriteKey = 'TEWEu8XrcMVejk8GOulbEx7rHGyuuijV';
+const testWriteKey = 'WJq9vAlUO5l2255jMg7eEthbkDtq1svu';
```

Comment:
This is a different test project that uses default configurations. The other one has some integration(s) turned on using dummy settings.